src/workerd/jsg/fast-api.h

```diff
@@ -17,10 +17,21 @@
 #include <v8-value.h>
 
 #include <kj/common.h>
+#include <kj/string.h>
```

Comment:
I think you could forward declare it:

```suggestion
namespace kj {
  class String;
}
```